lib/geocoding.rb

```diff
@@ -1,13 +1,12 @@
 class Geocoding
   attr_reader :params, :location
 
-  def initialize(location, params = { params: { region: 'uk' } })
```

Comment:
Does OS support Northern Ireland? I know we don't support schools there, but is it something that we'd need to be aware of if we do?